test/unary_test.js

```diff
@@ -670,7 +674,7 @@ describe('test unary', function() {
           0.21339342411295945,
           -0.6558230571220807,
         ],
-        [3, 2, 2, 1]);
+        [3, 2, 2, 1], 1 / 1024);
```

Comment:
(nit) Would feel cleaner/more consistent on a separate line like the other entities.

```suggestion
        [3, 2, 2, 1],
        1 / 1024);
```
Though, if a code reformatting tool is the cause of this, then sigh, never mind.